app/views/test/index.html.erb

```diff
@@ -1,2 +1,212 @@
-<p>This is a test page. It will serve as the home page for logged in users.</p>
-<p>In the meantime, however, have a Weibo alien. <i class="fa fa-weibo"></i></p>
+<div class="col-md-8" style="padding:0px;">
+  <div class="panel panel-default" style="border-radius:15px;margin-left:10px;margin-right:10px;">
+    <div class="panel-heading" style="border-top-left-radius:15px;border-top-right-radius:15px;"><b>Videos In Progress</b> <a href="#" style="float:right;">+</a></div>
+    <table class="table" style="margin-left:7px;margin-right:7px;padding-bottom:10px;">
+      <thead>
+        <tr>
+          <th>Title</th>
+          <th>Category</th>
+          <th>Duration</th>
+          <th>Completed</th>
+          <th>Review</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><a href="#">Area of a Shaded Region</a></td>
+          <td>Math</td>
+          <td>1:31</td>
+          <td>11%</td>
+          <td><button class="btn btn-primary btn-xs">Submit</button></td>
+        </tr>
+        <tr>
+          <td><a href="#">Area of an Equilateral Triangle</a></td>
+          <td>Math</td>
+          <td>3:42</td>
+          <td>76%</td>
+          <td><button class="btn btn-primary btn-xs">Submit</button></td>
+        </tr>
+        <tr>
+          <td><a href="#">Area of a Sector Given an Angle</a></td>
+          <td>Math</td>
+          <td>2:26</td>
+          <td>30%</td>
+          <td><button class="btn btn-primary btn-xs">Submit</button></td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+  <div class="panel panel-default" style="border-radius:15px;margin-left:10px;margin-right:10px;padding-bottom:10px;">
+    <div class="panel-heading" style="border-top-left-radius:15px;border-top-right-radius:15px;"><b>Videos Recently Translated</b> <a href="#" style="float:right;">+</a></div>
+    <table class="table" style="margin-left:7px;margin-right:7px;">
+      <thead>
+        <tr>
+          <th>Title</th>
+          <th>Category</th>
+          <th>Duration</th>
+          <th>Completed</th>
+          <th>Review</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><a href="#">Combining Like Terms 1</a></td>
+          <td>Math</td>
+          <td>3:23</td>
+          <td>100%</td>
```

Comment:
In lieu of this percentage number, we could add another cute little progress bar. Just a suggestion.